kano_settings/system/advanced.py

```diff
@@ -136,8 +138,60 @@ def add_safesearch_blacklist(hosts):
         ]
 
     for subdomain in second_domains:
-        hosts.append(url_pattern.format(subdomain))
-        hosts.append(www_pattern.format(subdomain))
+        hosts.append(url_pattern.format('google.{}'.format(subdomain)))
+        hosts.append(www_pattern.format('google.{}'.format(subdomain)))
+
+    # Block additional search sites
+    additional_search_sites=[
+        'bing.com',
+        'search.yahoo.com',
+        'uk.search.yahoo.com',
+        'ask.com',
+        'uk.ask.com', # pycountry does not return "uk", but "gb"
+        'search.aol.com',
+        'aolsearch.com',
+        'wow.com',
+        'webcrawler.com',
+        'zoo.com', # Webcrawler sometimes redirects to zoo.com
+        'mywebsearch.com',
+        'home.mywebsearch.com',
+        'infospace.com',
+        'info.com',
+        'duckduckgo.com',
+        'blekko.com',
+        'contenko.com',
+        'dogpile.com',
+        'alhea.com'
+        ]
+
+    # Add the additional search sites
+    for additional_site in additional_search_sites:
+        hosts.append(url_pattern.format(additional_site))
+        hosts.append(www_pattern.format(additional_site))
+
+    # Add subdomains to the sites that need it
+    for country in pycountry.countries:
+        hosts.append (url_pattern.format('{}.ask.com'.format(country.alpha2.lower())))
```

Comment:
Sorry to be annoying but can you create a function out of this code? 
```
def myfunc(mywebsite):
  hosts.append (url_pattern.format('mywebsite'.format(country.alpha2.lower())))
  hosts.append (www_pattern.format('mywebsite'.format(country.alpha2.lower())))
```

And add all the search engines into a list, that way it's much easier if in the future we need to add new ones